CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ For details about compatibility between different releases, see the **Commitment
 
 - Fix `last activity` not updating when an end device joins for the first time in the Console.
 - Fix a bug that would show the "Status count periodicity"-field in the Console as `200` when actually set to `0`.
+- Fix random Cypress test failures when logging in programmatically.
```

Comment:
This is not relevant for the changelog, since it only concerns development and is not affecting users in any way.